airbyte/progress.py

```diff
@@ -129,13 +140,38 @@ def __init__(
         self.last_update_time: float | None = None
 
         self._rich_view: RichLive | None = None
+
+        self.reset_progress_style(style)
+
+    def reset_progress_style(
+        self,
+        style: ProgressStyle = ProgressStyle.AUTO,
+    ) -> None:
```

Comment:
https://github.com/coderabbitai generate unit tests for this method. Especially important to cover mocks for scenarios that branch logic into different directions.